### PR TITLE
fix: Complete GroupChannelList props interface

### PR DESCRIPTION
**Pull request by HoonBaek**
### Fix
* Use uikit-tools 0.0.1-alpha.44
* Update the `GroupChannelListQueryParamsInternal` with the recent supporting types
* Call `onChannelSelect` in the GroupChannelList
* Add a `channelListQuery` to the GruopChannelList props
* Add the `onChannelCreate` props to the CreateGroupChannel component
* Use the `onChannelCreate` props to set the current channel in the Custom App of storybook sample

### CODE REVIEW

File: src/modules/ChannelList/context/ChannelListProvider.tsx [hunk 1]
```diff
@@ -37,26 +44,21 @@ export interface ApplicationUserListQueryInternal {
...
-  hiddenChannelFilter?:
-    | 'all'
-    | 'unhidden_only'
-    | 'hidden_only'
-    | 'hidden_allow_auto_unhide'
-    | 'hidden_prevent_auto_unhide';
-  unreadChannelFilter?: 'all' | 'unread_message';
+  memberStateFilter?: MyMemberStateFilter;
+  hiddenChannelFilter?: HiddenChannelFilter;
+  unreadChannelFilter?: UnreadChannelFilter;
```
Comment (bang9) [inline src/modules/ChannelList/context/ChannelListProvider.tsx:61]: ⭐️⭐️⭐️⭐️⭐️

File: src/modules/ChannelList/utils.ts [hunk 2]
```diff
@@ -133,7 +134,7 @@ const createChannelListQuery = ({
   const params: GroupChannelListQueryParamsInternal = {
     includeEmpty: false,
     limit: 20, // The value of pagination limit could be set up to 100.
-    order: 'latest_last_message', // 'chronological', 'latest_last_message', 'channel_name_alphabetical', and 'metadata_value_alphabetical'
+    order: GroupChannelListOrder.LATEST_LAST_MESSAGE, // 'chronological', 'latest_last_message', 'channel_name_alphabetical', and 'metadata_value_alphabetical'
```
Comment (bang9) [inline src/modules/ChannelList/utils.ts:137]: We can remove `// 'chronological', 'latest_last_message', 'channel_name_alphabetical', and 'metadata_value_alphabetical'`!

File: src/modules/GroupChannelList/context/GroupChannelListProvider.tsx [hunk 3]
```diff
@@ -33,15 +44,16 @@ interface GroupChannelListContextType {
 
   // Custom
   // Partial props - because we are doing null check before calling these functions
+  channelListQuery?: GroupChannelListQueryParamsInternal;
```
Comment (bang9) [inline src/modules/GroupChannelList/context/GroupChannelListProvider.tsx:47]: How about adding TODO marks here?

Comment (HoonBaek) [reply src/modules/GroupChannelList/context/GroupChannelListProvider.tsx:47]: Good idea

File: src/modules/GroupChannelList/index.tsx
Comment (bang9) [inline src/modules/GroupChannelList/index.tsx:1]: nice catch thanks!

**Review by bang9 — APPROVED** [review]
lgtm~!

**Pull request merged**